pinot-plugins/pinot-input-format/pinot-orc/pom.xml

```diff
@@ -74,5 +74,9 @@
       <groupId>org.apache.orc</groupId>
       <artifactId>orc-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+    </dependency>
```

Comment:
Is this required?

Reply:
Yes. Without it, we will get the exception `com.google.protobuf.MessageOrBuilder` not found